packages/flame_forge2d/lib/world_contact_listener.dart

```diff
@@ -0,0 +1,79 @@
+import 'flame_forge2d.dart';
+
+/// Listens to the entire [World] contacts events.
+///
+/// It propagates contact events ([beginContact], [endContact], [preSolve],
+/// [postSolve]) to other [ContactCallbacks]s when a [Body] or at least one of
+/// its fixtures `userData` is set to a [ContactCallbacks].
+///
+/// {@template flame_forge2d.world_contact_listener.algorithm}
+/// If the [Body] `userData` is set to a [ContactCallbacks] the contact events
+/// of this will be called as well when any [Body]'s fixture contacts another
+/// [Fixture].
```

Comment:
```suggestion
/// If the [Body] `userData` is set to a [ContactCallbacks] the contact events
/// of this will be called when any [Body]'s fixture contacts another [Fixture].
```